ddpui/schemas/dashboard_schema.py

```
@@ -63,6 +77,7 @@ class DashboardResponse(Schema):
     filter_layout: str
     layout_config: list[dict]
     components: dict
+    tabs: List[DashboardTabSchema] = []  # Array of tabs
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)